### PR TITLE
Add `wrenfold` as a package

### PR DESCRIPTION
**Pull request by gareth-cross**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by gareth-cross** [comment]
@conda-forge/help-python, ready for review.

**Comment by gareth-cross** [comment]
Hi @xhochy, is there anything else I need to tag this with to start the review process? I just want to validate that I'm not missing a pre-requisite step.

File: recipes/wrenfold/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    - python
+
+test:
+  imports:
+    - wrenfold
+    - wrenfold.sym
+  requires:
+    - pip
+    - numpy
+    - sympy
```
Comment (xhochy) [inline recipes/wrenfold/meta.yaml:36]: Why are they required here?
```suggestion
    - numpy
    - sympy
```

Comment (gareth-cross) [reply recipes/wrenfold/meta.yaml:36]: The command `test_wheel.py` runs the unit tests + examples, which depend on `numpy` and `sympy`.

They are a hard dependency only at test time. At runtime they are optional dependencies.

**Comment by gareth-cross** [comment]
Thank you for the review @xhochy.

**Pull request merged**